notifications_api_common/kanalen.py

```diff
@@ -39,7 +47,28 @@ def __repr__(self):
             self.main_resource,
         )
 
-    def get_kenmerken(self, obj: Model, data: Dict = None) -> Dict:
+    @staticmethod
+    def get_field(model: Model, field: str) -> Field:
```

Comment:
Minor semantic comment, but should this be `field_name` or `field_str` instead of `field` since it is not a `Field`?

Reply:
yup, that makes more sense, I'll change it